main/src/main/java/org/mobilitydata/gtfsvalidator/validator/RouteAgencyIdValidator.java

```diff
@@ -18,43 +18,50 @@
 
 import javax.inject.Inject;
 import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
-import org.mobilitydata.gtfsvalidator.notice.MissingRequiredFieldNotice;
-import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
-import org.mobilitydata.gtfsvalidator.table.GtfsAgencyTableContainer;
-import org.mobilitydata.gtfsvalidator.table.GtfsRoute;
-import org.mobilitydata.gtfsvalidator.table.GtfsRouteTableContainer;
+import org.mobilitydata.gtfsvalidator.notice.*;
+import org.mobilitydata.gtfsvalidator.table.*;
 
 /**
  * Checks that agency_id field in "routes.txt" is defined for every row if there is more than 1
- * agency in the feed.
+ * agency in the feed, recommended if only 1 agency.
  *
  * <p>Generated notice: {@link MissingRequiredFieldNotice}.
+ *
+ * <p>Generated notice: {@link MissingRecommendedFieldNotice}.
  */
 @GtfsValidator
-public class TripAgencyIdValidator extends FileValidator {
+public class RouteAgencyIdValidator extends FileValidator {
```

Comment:
Did you actually delete `TripAgencyIdValidator`?  Or just fork it?  It seems like `TripAgencyIdValidator` could be deleted entirely?

Reply:
No, I'm not sure why it looks like that in the diff...  I dupblicated it because `agency_id` is required for `routes.txt`, `trips.txt` and `agency.txt` when there is more than one agency in `agency.txt` and recommended when there is only 1.

Reply:
(there was already a validator for this for `trips.txt`, that is what `TripAgencyIdValidator` did.  I added it for `routes.txt`)

Reply:
There is a TripAgencyIdValidator in the master branch: https://github.com/MobilityData/gtfs-validator/blob/master/main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TripAgencyIdValidator.java

So looks like you renamed it and the old file won't be around, which is +1 from me.